### PR TITLE
adds a medium test for plugins webservices

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: server/sonar-server/src/test/java/org/sonar/server/plugins/ws/PluginsWsMediumTest.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  }
+
+  private WsTester restartServerTester() {
+    WsTester wsTester;
+    serverTester.restart();
+    // correctly simulate a server restart (ie. user is disconnected)
+    userSessionRule.anonymous();
+    // must use a new WsTester to reference the right PluginWs instance
+    wsTester = new WsTester(serverTester.get(PluginsWs.class));
+    return wsTester;
```
Comment (jblievremont) [inline server/sonar-server/src/test/java/org/sonar/server/plugins/ws/PluginsWsMediumTest.java]: I would inline the instanciation and remove the declaration of the intermediate variable.

**Comment by jblievremont** [comment]
Very small feedback and LGTM!

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

File: server/sonar-server/src/test/java/org/sonar/server/plugins/ws/PluginsWsMediumTest.java (hunk 1 above)
Comment (sns-seb) [inline server/sonar-server/src/test/java/org/sonar/server/plugins/ws/PluginsWsMediumTest.java]: you're right

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

**Pull request merged**